app/controllers/api/subcollections/container_images.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module Api
+  module Subcollections
+    module ContainerImages
+      def container_images_scan_resource(provider_id, type = nil, image_id = nil, _payload = nil)
```

Comment:
Is there a reason all these defaults are here? Is somewhere that's calling this method with just the first arg?

Reply:
good point, fixing in revision